api/services/Reddit.js

```diff
@@ -5,24 +5,16 @@ var request = require("request"),
   userAgent = "Webpage:hq.porygon.co:v" + sails.config.version;
 
 exports.refreshToken = function (refreshToken, error, callback) {
-  var data = "client_secret=" + sails.config.reddit.clientIDSecret +
-    "&client_id=" + sails.config.reddit.clientID +
-    "&duration=permanent" +
-    "&state=fapprefresh" +
-    "&scope=identity" +
-    "&grant_type=refresh_token" +
-    "&refresh_token=" + refreshToken +
-    "&redirect_uri=" + sails.config.reddit.redirectURL;
+  var data = "grant_type=refresh_token&refresh_token=" + refreshToken;
```

Comment:
These changes aren't essential, but I was messing around with these parameters while debugging and realized that most of them are unnecessary.